Classes/Service/ResourceService.php

```diff
@@ -15,13 +15,13 @@ class ResourceService {
 	 * Appends a shortened md5 of the file.
 	 *
 	 * @param string $path The location of the resource, can be either a path relative to the Public resource directory of the package or a resource://... URI
-	 * @param string $package Target package key. If not set, the current package key will be used
+	 * @param null|string $package Target package key. If not set, the current package key will be used
 	 * @return string
 	 * @throws \Exception
 	 */
-    public function getResourceUri($path, $package = null)
+    public function getResourceUri(string $path, string $package = null): string
     {
-        if (strpos($path, 'resource://') === 0) {
+        if (str_starts_with($path, 'resource://')) {
```

Comment:
Flow 7 can also be installed with php 7.4 and this method is only available in php 8 and there is no polifill provided which adds this method for php 7. Please add the polifill!

Reply:
Danke @simonschaufi - wir haben das parallel zu deinem Feedback bereits gelöst, indem wir dieses Package klar als "nur für PHP 8.x" deklarieren, siehe https://github.com/mrimann/CacheBreaker/pull/6

Reply:
ok. Didn't see that.